src/components/sbb-tab-group/sbb-tab-group.stories.tsx

```diff
@@ -41,7 +41,7 @@ const tabPanelFour = (): JSX.Element => (
 );
 
 const DefaultTemplate = (args): JSX.Element => (
```

Comment:
Maybe change to `{size, ...args}`? Otherwise I think the size will be passed also in the `firstTabTitle` function

Reply:
And same for other templates